src/components/documentation/_page.scss

```diff
@@ -463,3 +467,74 @@
     margin-top: 3px;
 }
 
+.tui-editor-contents .endpoint.get:before {
```

Comment:
Shouldn't it be .method.get instead of .endpoint.get ?

Also available for the other verbs.